fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -379,4 +390,24 @@
     </field>
   </xsl:template>
 
+  <!-- add mods_note_Tags_ms -->
+  <xsl:template match="mods:mods/mods:note[@displayLabel='Tags']" mode="utk_MODS">
+    <field name="utk_mods_note_Tags_ms">
+      <xsl:value-of select="normalize-space(.)"/>
+    </field>
+  </xsl:template>
+
+  <!-- add mods_originInfo_ms -->
+  <xsl:template match="mods:mods/mods:originInfo[mods:dateCreated or mods:dateOther]" mode="utk_MODS">
+    <field name="utk_mods_originInfo_date_ms">
+      <xsl:value-of select="child::mods:*[contains(local-name(),'dateCreated') or contains(local-name(),'dateOther')]"/>
+    </field>
+  </xsl:template>
+
+  <xsl:template match="mods:mods/mods:physicalDescription[mods:form[@authority='aat']]" mode="utk_MODS">
```

Comment:
@CanOfBees - I believe lines 407 to 411 duplicate work done on lines 133 to 147, correct? We don't need to define form with an authority twice.

Reply:
@mlhale7 very good question (I'm not sure!:])

I was following the spreadsheet literally (except for the obvious XPath duplicates); the field generated at 407-411 is for `form_authority_aat`, vs the more generic `form_authority` or `form` fields generated at 134-147.

I think the `form_authority_aat` is a collection-specific requirement?

I'm happy to drop the extra t template if you think it's unnecessary. What are your thoughts?

Thanks!

Reply:
Other than in two places where a URI was mistakenly entered instead of an authority code (example: {'@authority': 'http://vocab.getty.edu/aat/300027275', '#text': 'corporation reports'}), the only authority we use on form is aat. This makes it so that currently just distinguishing form  by those with and without authorities is enough to also separate out all the aat terms. It also leaves it to be flexible if we ever do decide to add an additional vocabulary for standard form terms. If you could drop the extra template, I think that would be best. The spreadsheet did specify the aat specifically - sorry for not noticing that until now (I'll update it).

Reply:
Sounds great - thanks for the detailed response! I'll update the stylesheet.